ihor.bohdan/first/task 2/hash.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+hash = { 'Yes' => 23,
+         'b' => 'travel',
+         'yesterday' => 34,
+         5 => '234',
+         yesss: :fg,
+         try: 30,
+         key: 'some value',
+         ' yesterday1 ' => 34,
+         ' yesteryear ' => 2014 }
+
+puts(hash.select { |k, v| k =~ /^yes/ }.count)
```

Comment:
```suggestion
puts(hash.count { |k, v| k =~ /^yes/ })
```

Reply:
https://apidock.com/ruby/Enumerable/count
There is an ability to pass block to count so you will have to iterate over the hash only once

Reply:
https://github.com/rubocop-hq/ruby-style-guide#underscore-unused-vars
Also a good practise to use _ for an unused variables